backend/src/main/java/com/zzang/chongdae/auth/config/AuthWebMvcConfig.java

```diff
@@ -27,6 +27,11 @@ public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers)
     public void addInterceptors(InterceptorRegistry registry) {
         registry.addInterceptor(new AuthorizationInterceptor(cookieConsumer, jwtTokenProvider))
                 .addPathPatterns("/**")
-                .excludePathPatterns("/auth/login", "/auth/signup");
+                .excludePathPatterns("/auth/login", "/auth/signup",
+                        "/swagger-ui/**",
+                        "/swagger-ui.html",
+                        "/v3/api-docs/swagger-config",
+                        "/static/swagger-ui/openapi3.yaml"
```

Comment:
저는 포케 방식 좋아요~~~ 아니면 아래처럼 그냥 싹 다 개행하는 것도 괜찮을듯!~ 뭐든 굿

```suggestion
                .excludePathPatterns(
                        "/auth/login", 
                        "/auth/signup",
                        "/swagger-ui/**",
                        "/swagger-ui.html",
                        "/v3/api-docs/swagger-config",
                        "/static/swagger-ui/openapi3.yaml"
```